example/src/main/java/com/tokenautocomplete/TokenActivity.java

```diff
@@ -59,18 +62,11 @@ protected boolean keepObject(Person person, String mask) {
         completionView.setTokenListener(this);
         completionView.setTokenClickStyle(TokenCompleteTextView.TokenClickStyle.Select);
 
-
-        if (savedInstanceState == null) {
```

Comment:
Why did you remove this?  Please add more comments to your commits.  At some point we might need to rebase this against the original project.  So documenting where things changed is helpful at times

Reply:
This is a sample activity. Just for demo. Doesn't affect the library. So these lines add the chips onCreate of an activity, I didn't want that in the demo to test all the cases.

Reply:
for a third-party library best approach would be to comment it out and leave it there with a comment about why it is commented out.  this makes future rebases clearer.